manageiq-operator/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Build the manager binary
+FROM docker.io/library/golang:1.18 as builder
```

Comment:
Wonder if we can somehow avoid this go version here (trying to avoid having to change multiple places on upgrade)

Reply:
That would be nice, but I don't have a good solution at the moment.